apps/src/aichat/views/modelCustomization/constants.ts

```diff
@@ -11,7 +11,14 @@ export const SET_TEMPERATURE_STEP = 0.1;
 export const MAX_RETRIEVAL_CONTEXTS = 20;
 export const MAX_ASK_ABOUT_TOPICS = 10;
 
-export const MODEL_CARD_FIELDS_AND_LABELS: [
+export const MODEL_CARD_FIELDS_AND_LABELS: [keyof ModelCardInfo, string][] = [
+  ['description', 'Description'],
+  ['intendedUse', 'Intended Use'],
+  ['limitationsAndWarnings', 'Limitations and Warnings'],
+  ['testingAndEvaluation', 'Testing and Evaluation'],
+];
+
+export const ALL_MODEL_CARD_FIELDS_LABELS_ICONS: [
```

Comment:
Hmm, it would be ideal to not have to create a second array here. The 'ALL' version is technically correct; the previous version skipped `'exampleTopics'` since it wasn't a single string field and at the time we were just handling `'exampleTopics'` separately (see `ModelCardFields.tsx` for example). I wonder if we should just keep this full list and update the affected components to specifically skip the `'exampleTopics'` if it's already being rendered a different way?

Reply:
Sure - using one array and skipping 'exampleTopics' when needed sounds good. It's currently being used in the levelbuilder edit page. We may also add 'botName' to this list as well in a follow-up.